assets/logic.js

```diff
@@ -1,4 +1,29 @@
 $('.jsToggle').on('click', function(){
 	var target = $(this).data('toggle');
 	$(target).toggleClass('active bounceInDown');
+});
+
+$(document).ready(function(){
+	function getRandom(min, max) {
+		return Math.floor(Math.random() * (max - min + 1)) + min;
+	}			
+	var canvas = document.getElementById('canvasContainer');
+	canvas.width = window.innerWidth;
+	canvas.height = window.innerHeight;
+	
+	var context = canvas.getContext('2d');
+	var stars = canvas.width / 2;
+	var colorrange = [0,60,240];
+
+	for (var i = 0; i < stars; i++) {
+		var x = Math.random() * canvas.offsetWidth;
```

Comment:
This needs to finish with a , instead of ; or have vars for all your other declerations below.